vatvalidate/validate.py

```diff
@@ -0,0 +1,74 @@
+from math import ceil
+
+from vatvalidate.exceptions import InvalidVATDigitsError
+
+
+def _modulus_9755(
+    weighted_digit_sum: int, vat_digits: list[int], use_9755: bool = False
+) -> bool:
+    """
+    Checks the weighted_digit_sum of the first 7 digits of a VAT number against the
+    last 2 digits of the VAT number to determine if the VAT number is valid, using
+    either the modulus 97 algorithm or the modulus 9755 algorithm.
+    """
+    # Add 55 to the weighted digit sum if we are using the 9755 algorithm
+    if use_9755:
+        weighted_digit_sum += 55
+
+    # Subtract 97 until we get a negative number, then take the absolute value
+    weighted_digit_sum = abs(weighted_digit_sum - 97 * ceil(weighted_digit_sum / 97))
```

Comment:
Is this different from the built in modulo in python? So just `weighted_digit_sum % 97` will give you the remainder of dividing by 97 although based on the comment maybe you want `97 - (weighted_digit_sum % 97)`?

Reply:
haha, yes!